app/src/main/java/org/desarrolladorslp/teckersapp/model/User.kt

```diff
@@ -0,0 +1,23 @@
+package org.desarrolladorslp.teckersapp.model
+import android.net.Uri
+
+class User(var Username: String?,var givenNameUser:String?, var familyNameUser:String?,var emailUser:String?,var idUser:String?, var photoUrlUser:Uri?) {
```

Comment:
All variables must start with a lowercase character, in this case it would be `username`